### PR TITLE
add a ci test

### PR DESCRIPTION
**Pull request by rubys**
Run ` ./vendor/bin/pest` after every push

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Run tests
+ 
+on: [push]
```
Comment (KTanAug21) [inline .github/workflows/ci.yml:3]: Can you also include the workflow to run when a pull request is created?  That way the user creating the pull request can immediately see the test run for their PR, and ultimately see if the PR passes or not

Comment (KTanAug21) [reply .github/workflows/ci.yml:3]: Haven't tested this: https://stackoverflow.com/questions/70964103/trigger-github-workflow-on-pull-request

**Review by KTanAug21 — APPROVED** [review]
This is great! Just added an additional feature that can be added, but totally optional!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rubys) [reply .github/workflows/ci.yml]: done

**Pull request merged**